packages/common-frontend/src/features/engine/slice.ts

```diff
@@ -92,7 +92,7 @@ export const syncNote = createAsyncThunk(
       return resp;
     }
     const data = resp.data!;
-    logger.info({ state: "pre:setNotes" });
+    logger.info({ state: "pre:setNotes", notes: data.map(NoteUtils.toLogObj) });
```

Comment:
this could be a lot of notes and might have performance impact. can we check perf of mapping 10k notes like this? 

also would change this to a debug statement

Reply:
This doesn't get used for note initialization I think, but I added a check to only log the notes if there's less than 10 notes. I think that does what I wanted, which is being able to see if the notes updated during development, and shouldn't affect performance for release.